releng/org.eclipse.xtext.dev-bom/pom.xml

```diff
@@ -65,8 +65,8 @@
 		<asm-version>7.1</asm-version>
 		<classgraph-version>4.8.35</classgraph-version>
 		
-		<error_prone_annotations-version>2.0.15</error_prone_annotations-version>
-		<guava-version>21.0</guava-version>
+		<error_prone_annotations-version>2.2.0</error_prone_annotations-version>
+		<guava-version>27.1-jre</guava-version>
```

Comment:
This can be overridden locally in case someone wants to use the android port, right?

Reply:
that was my assumption

Reply:
👍